index.html

```diff
@@ -5,21 +5,22 @@
 <body>
   <h1>Kiali - Observability for the service mesh</h1>
 <h2>tl;dr:</h2>
-Kiali provides answers to the question "What are my microservices in my Istio service mesh doing?"
+Kiali provides answers to the question "What are the microservices in my Istio service mesh doing?"
 
 <h2>Description of Kiali</h2>
-Microservice Architectures break up the monolith in many smaller pieces
+Microservice Architecture breaks up the monolith to many smaller pieces
```

Comment:
I think if you use a singular Architecture, the the phase needs to be prefixed with 'A ' (?)

perhaps in and to -> into ?